src/lib/templateengine.js

```diff
@@ -274,33 +276,20 @@ function TemplateEngine( undefined ) {
   
   this.enableAddressQueue = $.getUrlVar('enableQueue') ? true : false;
   
-  this.backend = 'cgi-bin'; // default path to backend
+  this.backend = 'default';
+  this.backendUrl = null;
```

Comment:
Thats the link I read before posting ;-) It sais there:

```
You might consider undefined to represent a system-level, unexpected, or error-like absence of value and null to represent program-level, normal, or expected absence of value.
```
Thats why I prefered to go with 'null', but this is rather an academical discussion and because it fits the cometvisu coding style better I will remove the null's